.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -7,7 +7,7 @@ name: Linux CI (!{{ build_environment }})
 
 on:
   # TODO: Enable pull_request builds when we can verify capacity can be met by auto-scalers
-{%- if on_pull_request %}
+{%- if on_pull_request or is_scheduled %}
```

Comment:
Assuming this isn't a permanent change right?

Reply:
Yes this is on a separate commit to test it